### PR TITLE
AC: Added support for ssd-vgg model

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

File: tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/README.md [hunk 1]
```diff
@@ -481,3 +481,6 @@ AccuracyChecker supports following set of adapters:
   * `output_name` - name of output node to convert.
 * `mask_to_binary_classification` - converts output of model represented as segmentation mask to `ArgMaxClassificationPrediction`. Class label calculated as comparision maximal probability in mask with given threshold.
   * `threshold` - probability threshold for label 1 (Optional, default 0.5).
+* `ssd_vgg16` - converting output of VGG based SSD model to `DetectionPrediction` representation.
```
Comment (eaidova) [inline tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/README.md:484]: not sure that it is connveted with vgg16 backbone, this repo provide also other ssd implementation, possibly we need give more general name and more description why these adapter should be used

File: tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/detection.py [hunk 2]
```diff
@@ -933,3 +933,69 @@ def process(self, raw, identifiers, frame_meta):
...
+            'boxes_out': StringField(description='boxes output')
+        })
+        return params
+
+    def configure(self):
+        self.scores_out = self.get_value_from_config('scores_out')
+        self.boxes_out = self.get_value_from_config('boxes_out')
+        self.confidence_threshold = 0.01
+        self.iou_threshold = 0.45
+        self.keep_top_k = 200
```
Comment (eaidova) [inline tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/detection.py:955]: please provide opportunity to set these parameters via config

File: tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/detection.py [hunk 3]
```diff
@@ -933,3 +933,69 @@ def process(self, raw, identifiers, frame_meta):
             results.append(DetectionPrediction(identifier, labels, scores, x_mins, y_mins, x_maxs, y_maxs))
 
         return results
+
+
+class SSDVGG16Adapter(Adapter):
+    __provider__ = 'ssd_vgg16'
```
Comment (eaidova) [inline tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/detection.py:939]: BTW, we have additional file for SSD adapters, please move this adapter to ssd.py

File: tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/README.md (hunk 1 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/README.md]: Done

File: tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/detection.py (hunk 2 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/detection.py]: Done

File: tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/detection.py (hunk 3 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/detection.py]: Done

**Review by eaidova — APPROVED** [review]

**Pull request merged**